weatherman.py

```diff
@@ -0,0 +1,80 @@
+"""
+Generating reports of temperature from given text files.
+"""
+
+import os
+import argparse
+import sys
+
```

Comment:
Line spacing after imports(PEP-8)

Reply:
Couldn't find anything on line spaces after imports in PEP-8. But did find 2 spaces before and after functions, which makes 2 spaces after imports, so did that. Let me know if I missed something here.